src/pages/Home/sections/NotableUsersSection.js

```diff
@@ -115,8 +115,7 @@ const NotableUsersSection = () => {
           <NavButton
             color='primary'
             href={{
-              pathname: '/organizations/affiliated',
-              query: { contributor: true },
+              pathname: '/organizations/contributors',
```

Comment:
Looks good.  Just a minor update to code style
`
  const MoreContributors = () => {
    return (
      <Grid item xs={12} style={{ padding: '30px' }}>
        <div align='center'>
          <NavButton
            color='primary'
            href={{ pathname: '/organizations/contributors' }}
          >
            View contributors
          </NavButton>
        </div>
      </Grid>
    );
  };
`